.github/workflows/e2e.yml

```diff
@@ -42,6 +42,10 @@ jobs:
           allowed-conclusions: success
           verbose: false # change for debugging
 
+      - name: Create kyma-system namespace
+        run: |
+          kubectl create namespace kyma-system
```

Comment:
shouldn't this be part of the make target? because, if you now follow the deploy instructions in the readme and use the deploy target, you will run into the same problem.

Reply:
Will be addressed in a separate PR.